scripts/Deepcell_Preprocessing.ipynb

```diff
@@ -45,15 +45,29 @@
     "# load TIFs from GUI-based directory structure\n",
     "data_dir = \"../data/example_dataset/input_data/\"\n",
     "\n",
+    "# data format\n",
+    "MIBItiff = False\n",
+    "MIBItiff_suffix = \"-MassCorrected-Filtered.tiff\"\n",
+    "\n",
+    "# nuclear channel name(s) (or None)\n",
+    "nucs = ['HH3']\n",
+    "\n",
+    "# membrane channel name(s) (or None)\n",
+    "mems = ['Membrane']\n",
+    "\n",
     "# validate data_dir\n",
     "data_utils.validate_paths(data_dir)\n",
     "\n",
     "# either get all points in the folder\n",
     "points = os.listdir(data_dir)\n",
     "points = [point for point in points if os.path.isdir(data_dir + point) and point.startswith(\"Point\")]\n",
+    "if MIBItiff:\n",
+    "    points = [point for point in points if point.endswith(MIBItiff_suffix)]\n",
```

Comment:
Feel free to chime in on this @ngreenwald, but I think it would be nice to split up some of these cells. For example, we have the MIBItiff variable assignments in one cell, the nuclear and membrane channel variable assignments in another, the validation of data_dir in another, etc. It can make the process easier to follow and potentially help with isolating buggy code.